src/main/java/com/googlecode/lanterna/gui2/ImageComponent.java

```diff
@@ -0,0 +1,54 @@
+package com.googlecode.lanterna.gui2;
+
+import com.googlecode.lanterna.*;
+import com.googlecode.lanterna.graphics.*;
+
+/**
+ * 
+ * @author ginkoblongata
+ */
+public class ImageComponent extends AbstractInteractableComponent {
+    
+    private TextImage textImage;
+    
+    public ImageComponent() {
+        setTextImage(new BasicTextImage(0,0));
+    }
+    
+    public void setTextImage(TextImage textImage) {
+        this.textImage = textImage;
+        invalidate();
+    }
+    
+    @Override
+    public InteractableRenderer<ImageComponent> createDefaultRenderer() {
+        return new InteractableRenderer<ImageComponent>() {
+            @Override
+            public void drawComponent(TextGUIGraphics graphics, ImageComponent panel) {
+                graphics.drawImage(TerminalPosition.TOP_LEFT_CORNER, textImage);
+            }
+            @Override
+            public TerminalSize getPreferredSize(ImageComponent panel) {
+                return textImage.getSize();
+            }
+            @Override
+            public TerminalPosition getCursorLocation(ImageComponent component) {
+                // when null, lanterna hidden cursor for this component
+                return null;
+            }
+        };
+    }
+    
+    @Override
+    public Result handleKeyStroke(com.googlecode.lanterna.input.KeyStroke keyStroke) {
```

Comment:
Why is this component interactable? Seems like it shouldn't be, like the existing EmptySpace?

Reply:
Generally for longer term, future version I would try to make the case that there should not be any non interactable components.  It makes the library more complex to have this distinction and I got hung up on it and no doubt the same can happen to other users.
I think some breaking change version should converge interactable down into the Component.
In the meantime, I'd prefer to always extend Interactable to lessen the future work to make such a simplification of the library.

Specifically though, users would like to make a mouse listener and when not interactable, the image component is much less useful.